src/tests/spec/corpus.rs

```diff
@@ -172,26 +273,6 @@ fn run_test(test: TestFile) {
             }
         }
 
-        // native_to_bson( bson_to_native(dB) ) = cB
-
```

Comment:
these were shifted earlier so that they can be run even without the decimal128 flag.